apps/vr-tests-react-components/src/stories/RatingDisplay.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { CircleRegular, CircleFilled, SquareRegular, SquareFilled } from '@fluentui/react-icons';
+import { RatingDisplay } from '@fluentui/react-rating-preview';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('RatingDisplay Converged Size', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addStory('RatingDisplay size small', () => <RatingDisplay size="small" value={3.5} />, {})
```

Comment:
It's fine to have just one story per size (the value and count one), and not a value-only story for each size.
```suggestion
```